### PR TITLE
feat: manager kafka cluster in sqlness runner

### PR DESCRIPTION
**Pull request by waynexia**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

Makes it easier to run sqlness using kafka wal. See `tests/README.md` for usage.

## Checklist

- [x]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.
- [x]  This PR does not require documentation updates.

### CODE REVIEW

**Review by WenyXu — APPROVED** [review]

File: tests/runner/src/util.rs [hunk 1]
```diff
@@ -119,3 +120,57 @@ pub async fn check_port(ip_addr: SocketAddr, timeout: Duration) -> bool {
...
+            "kafka-cluster.yml",
+            "up",
+            "kafka",
+            "-d",
+            "--wait",
+        ])
+        .output()
+        .expect("Failed to start kafka cluster");
+
+    println!("kafka cluster is on");
```
Comment (niebayes) [inline tests/runner/src/util.rs:156]:
```suggestion
    println!("kafka cluster is up");
```

Comment (niebayes) [reply tests/runner/src/util.rs:156]: Since the command is `docker-compose ... up/down`.

File: tests/README.md [hunk 2]
```diff
@@ -23,6 +23,20 @@ modes. E.g., `standalone/` contains all the tests to run under `greptimedb stand
 Under the first level of subdirectory (e.g. the `cases/standalone`), you can organize your cases as you like.
 Sqlness walks through every file recursively and runs them.
 
+### Kafka WAL
+
+Sqlness supports Kafka WAL. You can either provide a Kafka cluster or let sqlness to start one for you.
+
+To run test with kafka, you need pass the option `-w kafka`. If no other options are provided, sqlness will use `conf/kafka-cluster.yml` to start a Kafka cluster. This requires `docker` and `docker-compose` commands in your environment.
```
Comment (niebayes) [inline tests/README.md:30]:
```suggestion
To run test with kafka, you need to pass the option `-w kafka`. If no other options are provided, sqlness will use `conf/kafka-cluster.yml` to start a Kafka cluster. This requires `docker` and `docker-compose` commands in your environment.
```

File: tests/README.md [hunk 3]
```diff
@@ -23,6 +23,20 @@ modes. E.g., `standalone/` contains all the tests to run under `greptimedb stand
...
+
+Otherwise, you can additionally pass the your existing kafka environment to sqlness with `-k` option. E.g.:
+
+```shell
+cargo sqlness -w kafka -k localhost:9092
+```
+
+In this case, sqlness will not start its own kafka cluster and the one you provided instead.
```
Comment (niebayes) [inline tests/README.md:38]:
```suggestion
In this case, sqlness will not start its own kafka cluster and start the one you provided instead.
```

**Review by niebayes — APPROVED** [review]

File: tests/README.md (hunk 3 above)
Comment (waynexia) [reply tests/README.md:38]: No, it will use the provided endpoint

**Pull request merged**